resources/sdk/purecloudgo/templates/model.mustache

```diff
@@ -15,6 +16,45 @@ type {{classname}} struct { {{#vars}}
 
 {{/vars}}}
 
+func (u *{{classname}}) MarshalJSON() ([]byte, error) {
+	// Redundant initialization to avoid unused import errors for models with no Time values
+	_  = timeutil.Timedelta{}
+	type Alias {{classname}}
+
+	{{#vars}}{{#isDateTime}}
+	{{name}} := new(string)
+	if u.{{name}} != nil {
+		{{#vendorExtensions.x-local-date-time}}*{{name}} = timeutil.Strftime(u.{{name}}, "%Y-%m-%dT%H:%M:%S.%f"){{/vendorExtensions.x-local-date-time}}
```

Comment:
Are these valid if there's no milliseconds? There are 4 known formats used by the Java SDK: https://github.com/purecloudlabs/platform-client-sdk-common/blob/master/resources/sdk/purecloudjava/extensions/ApiDateFormat.java#L14.

Reply:
Any of them. I believe it's an API-layer behavior that truncates the milliseconds when they're exactly zero.

Reply:
That should be ok then. This change was more for formatting times to be sent to the API as a request payload.
I doubt there would be any issues arising from a client expecting `2021-08-04T08:00:00Z`  and receiving `2021-08-04T08:00:00.00Z` instead.

Reply:
AFAIK that's accurate. I believe the API is tolerant of optional ms.